packages/signals/signals/src/core/client/index.ts

```diff
@@ -69,10 +75,25 @@ export class SignalsIngestClient {
     return analytics
   }
 
+  private shouldIngestSignals(): boolean {
```

Comment:
I don't think we don't want to calculate this on a per event basis -- see comment above.